jenkins/jenkins2-ha-agents.yaml

```diff
@@ -2208,6 +2220,55 @@ Resources:
     Properties:
       ThroughputInMibps: !Ref EFSProvisionedThroughputInMibps
       ServiceToken: !GetAtt 'LambdaFunction.Arn'
+  BackupVault:
+    Type: 'AWS::Backup::BackupVault'
+    Properties:
+      # AccessPolicy: # FIXME later
+      BackupVaultName: !Ref 'AWS::StackName'
+      # EncryptionKeyArn: # FIXME later
```

Comment:
@michaelwittig Can we fix that now? Otherwise, this should be documented in the *limitations* section?

Reply:
fixed